scripts/changelog

```diff
@@ -7,7 +7,7 @@ else
   if [ "$1" == "--pending" ]; then
     github_changelog_generator -t $GITHUB_TOKEN --unreleased-only -o "$2"
   elif [ "$1" == "--release-tag" ]; then
-    github_changelog_generator -t $GITHUB_TOKEN --unreleased true --unreleased-label "$2"
+    github_changelog_generator -t $GITHUB_TOKEN --unreleased true --future-release "$2"
```

Comment:
This should have always been `--future-release` instead of `--unreleased-label`. I only recently discovered the difference.